src/site/content/en/fast/content-delivery-networks/index.md

```diff
@@ -0,0 +1,284 @@
+
+
+### Compression
+
+All text-based responses should be compressed with either gzip or Brotli. If you have the choice, choose Brotli over gzip. Brotli is a newer compression algorithm, and compared to gzip, it can achieve higher compression ratios.  
+
+There are two types of CDN support for Brotli compression: "Brotli from origin" and "automatic Brotli compression".
+
+
+#### Brotli from origin
+
+Brotli from origin is when a CDN serves resources that were Brotli-compressed by the origin. Although this may seem like a feature that all CDNs should be able to support out of the box, it requires that a CDN be able to cache multiple versions (in other words, gzip-compressed and Brotli-compressed versions) of the resource corresponding to a given URL.
+
+
+#### Automatic Brotli compression
+
+Automatic Brotli compression is when resources are Brotli compressed by the CDN. CDNs can compress both cacheable and non-cacheable resources.
+
+The first time that a resource is requested it is served using "good enough" compression - for example, Brotli-5. This type of compression is applicable to both cacheable and non-cacheable resources.
+
+Meanwhile, if a resource is cacheable, the CDN will use offline processing to compress the resource at a more powerful but far slower compression level - for example, Brotli-11. Once this compression completes, the more compressed version will be cached and used for subsequent requests.
+
+
+#### Compression best practices
+
+Sites that want to maximize performance should apply Brotli compression at both their origin server and CDN. Brotli compression at the origin minimizes the transfer size of resources that can't be served from the cache. To prevent delays in serving requests, the origin should compress dynamic resources using a fairly conservative compression level - for example, Brotli-4; static resources can be compressed using Brotli-11.
+
+
+### TLS 1.3
+
+TLS 1.3 is the newest version of [Transport Layer Security (TLS)](https://en.wikipedia.org/wiki/Transport_Layer_Security), the cryptographic protocol used by [HTTPS](https://en.wikipedia.org/wiki/HTTPS). TLS 1.3 provides better privacy and performanced compared to TLS 1.2
+
+TLS 1.3. shortens the TLS handshake from two roundtrips to one. For connections using HTTP/1 or HTTP/2, shortening the TLS handshake to one roundtrip effectively reduces connection setup time by 33%.
+
+<figure class="w-figure">
+  <img src="./cdn3.png" alt="Comparison of the TLS 1.2 and TLS 1.3 handshakes" class="w-screenshot">
+</figure>
+
+### HTTP/2 & HTTP/3
+
+HTTP/2 and HTTP/3 both provide performance benefits over HTTP/1. Of the two, HTTP/3 offers greater _potential_ performance benefits. HTTP/3 isn't fully standardized yet, but it will be widely [supported](https://caniuse.com/#feat=http3) once this occurs.
```

Comment:
Is there a reason that we are making this strong claim about HTTP/3? It seems risky to say "it **will** be supported once it gets fully standardized" especially since there are factors outside of our control (other browser vendors)

Reply:
This should be OK. H3 is in all major browsers at the moment - albeit behind a flag. My original wording was less strong than this but it was suggested that this would be more accurate.